index.html

```diff
@@ -1,109 +1,114 @@
 <!DOCTYPE html>
 <html lang="en">
 
-    <!DOCTYPE html>
-    <html lang="en">
-    <head>
-        <meta charset="UTF-8">
-        <meta http-equiv="X-UA-Compatible" content="IE=edge">
-        <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <title>to do</title>
-        <!-- Main Template Css File-->
-        <link rel="stylesheet" href="todo.css">
-        <!-- Font Awesome Library-->
-        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta2/css/all.min.css" integrity="sha512-YWzhKL2whUzgiheMoBFwW8CKV4qpHQAEuvilg9FAn5VJUDwKZZxkJNuGM4XkWuk94WCrrwslk8yWNGmY1EduTA==" crossorigin="anonymous" referrerpolicy="no-referrer"/>
-        <!-- Render All Elements Normally-->
-        <link rel="stylesheet" href="normalize.css">
-        <!-- Google Fonts-->
-        <link rel="preconnect" href="https://fonts.googleapis.com">
-        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-        <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500;600;700&display=swap" rel="stylesheet">
-    </head>
-<header class="header">
-    <div class="toggle">
-        <div class="darkmode">
-            <input type="checkbox" class="checkbox" id="checkbox">
-            <label for="checkbox" class="lable1"></label>
-            <div class="ball"></div>
-        </div>
-    </div>
-    <div>
-        <h1 class="page-name">TO DO LIST</h1>
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
These lines already exist (duplicate)

<img width="553" alt="Screen Shot 2021-11-14 at 10 47 22 PM" src="https://user-images.githubusercontent.com/47992412/141698012-c65fb82d-2860-4a53-b5b7-915e6dfaf456.png">

Reply:
good catch! somehow I missed it, fixed now